### PR TITLE
Streaming example

### PR DESCRIPTION
**Pull request by judepereira**

### CODE REVIEW

File: examples/http/streaming/README.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    events:
+      help: Subscribe to events
+      event:
+        arguments:
+          name:
+            type: string
+            required: true        
+      http:
+        method: get
+        endpoint: /subscribe
```
Comment (stevepeak) [inline examples/http/streaming/README.md]: Different subscribe and unsubscribe endpoints.

**Comment by stevepeak** [comment]
This pull has become stale, sorry about that. There are multiple things that need to be updated according to new specs. Let's discuss more in Slack

File: examples/http/streaming/README.md (hunk 1 above)
Comment (judepereira) [reply examples/http/streaming/README.md]: Yes, I'll update this one.

Comment (judepereira) [reply examples/http/streaming/README.md]: @stevepeak Can you check now?

**Comment by judepereira** [comment]
Closing, will be handled by #26

**Pull request closed without merging**